src/DynamoCore/DynamoCore.csproj

```diff
@@ -123,7 +123,17 @@ limitations under the License.
     <Compile Include="Models\RunSettings.cs" />
     <Compile Include="Core\Threading\AsyncTaskExtensions.cs" />
     <Compile Include="Interfaces\IPathResolver.cs" />
+    <Compile Include="PackageManager\Interfaces\IFileCompressor.cs" />
+    <Compile Include="PackageManager\Interfaces\IDirectoryInfo.cs" />
+    <Compile Include="PackageManager\Interfaces\IFileInfo.cs" />
+    <Compile Include="PackageManager\Interfaces\IFileSystem.cs" />
+    <Compile Include="PackageManager\MutatingFileCompressor.cs" />
+    <Compile Include="PackageManager\MutatingFileSystem.cs" />
+    <Compile Include="PackageManager\MutatingPathRemapper.cs" />
     <Compile Include="PackageManager\PackageManagerResult.cs" />
+    <Compile Include="PackageManager\PackageUploadBuilder.cs" />
+    <Compile Include="PackageManager\TrueDirectoryInfo.cs" />
+    <Compile Include="PackageManager\TrueFileInfo.cs" />
     <Compile Include="Search\Interfaces\ISearchCategory.cs" />
```

Comment:
Almost all of these new files are required for mocking purposes and should be very straightforward to inspect.